pkg/component/controller/etcd_member_reconciler.go

```diff
@@ -142,9 +142,10 @@ func (e *EtcdMemberReconciler) waitForCRD(ctx context.Context) error {
 	log.Info("waiting to see EtcdMember CRD ready")
 	return watch.FromClient[*crdList, crd](ec.CustomResourceDefinitions()).
 		WithObjectName(fmt.Sprintf("%s.%s", "etcdmembers", "etcd.k0sproject.io")).
-		WithErrorCallback(func(cbErr error) (retryDelay time.Duration, err error) {
-			if retryAfter, e := watch.IsRetryable(err); e == nil {
-				log.WithError(err).Infof(
+		WithErrorCallback(func(cbErr error) (time.Duration, error) {
+			retryAfter, e := watch.IsRetryable(cbErr)
+			if e == nil {
```

Comment:
By not unnesting this if statement, there's less confusion if `e` or `cbErr` should be used later on.